lib/rspec/core/formatters/exception_presenter.rb

```diff
@@ -31,8 +31,31 @@ def colorized_message_lines(colorizer=::RSpec::Core::Formatters::ConsoleCodes)
           end
         end
 
-        def formatted_backtrace
-          backtrace_formatter.format_backtrace(exception_backtrace, example.metadata)
+        def formatted_backtrace(exception = @exception)
+          lines = backtrace_formatter.format_backtrace((exception.backtrace || []), example.metadata)
+          potential_lines = []
+
+          while exception.respond_to?(:cause) && exception.cause
```

Comment:
Would I have to define this method rspec-core?`RSpec::Support::Ruby.supports_exception_cause?` 

I am guessing this is so rspec still works with older versions of ruby that don't support `Exception#cause`